strutil/strutil.go

```diff
@@ -231,6 +231,16 @@ func CommaSeparatedList(str string) []string {
 	return filtered
 }
 
+// MultiCommaSeparatedList parses each string in strs with CommaSeparatedList
+// and returns the concatenation of all parsed values.
+func MultiCommaSeparatedList(strs []string) []string {
```

Comment:
Nitpick, but I think it provides clearer intent:
```suggestion
func FlattenCommaSeparatedLists(strs []string) []string {
```

Reply:
I agree and prefer your approach to this naming and description, but this is an existing function in the ["github.com/canonical/x-go/strutil"](https://pkg.go.dev/github.com/canonical/x-go/strutil) package, so for compatibility's sake I don't think we should change them here, if the code body is otherwise unchanged.

Reply:
Ah, a bit annoying we have a copy of another package, but I see